lib/seccomp-tools/asm/compiler.rb

```diff
@@ -159,6 +178,16 @@ def eval_constants(str)
 
       attr_reader :token
 
+      # <goto|jmp|jump> <label|Integer>
+      def jmp_abs
+         tk = token.fetch('goto') ||
```

Comment:
Looks weird?
`tk` is not used

Reply:
yeah for that one I just want to fetch the goto/jmp/jump token for absolute jump but I probably am not doing anything with that.